spec/decorators/order_decorator_spec.rb

```diff
@@ -18,6 +18,27 @@
 
   it '#number' do
     allow(decorator).to receive(:id).and_return(1)
+
     expect(decorator.number).to eq('Order #1')
   end
+
+  describe '#status_to_s' do
+    it 'returns string In delivery' do
+      order.update(status: 2)
+
+      expect(decorator.status_to_s).to eq(I18n.t('order.statuses.in_delivery'))
+    end
+
+    it 'returns string Delivered' do
+      order.update(status: 3)
+
+      expect(decorator.status_to_s).to eq(I18n.t('order.statuses.delivered'))
+    end
+
+    it 'returns string Canceled' do
+      order.update(status: 4)
+
+      expect(decorator.status_to_s).to eq(I18n.t('order.statuses.canceled'))
+    end
```

Comment:
It would be better when you will use fixed strings in your tests. In this case, you will test your decorator and dictionary (I18n)
Because now if I change `order.statuses.canceled` in `I18n` to `blah-blah-blah`, your test passes but your users will be disappointed